inkcpp_compiler/json_compiler.cpp

```diff
@@ -299,7 +299,7 @@ namespace ink::compiler::internal
 		else if (get(command, "CNT?", val))
 		{
 			// TODO: Why is this true again?
-			_emitter->write_path(Command::READ_COUNT, CommandFlag::NO_FLAGS, val, true);
+			_emitter->write_path(Command::READ_COUNT, CommandFlag::NO_FLAGS, val, false);
```

Comment:
I'm going to investigate this before I approve. Just to make sure...

Reply:
Okay, this actually needs to be true and the fact it's failing means there's another issue. With your change, the CNT? command actually won't work anymore because it's expecting an index in the container visits table, not the actual byte offset of the container in the instruction data. Can you create a new issue in GitHub with the failing file and the error message?